src/components/virtual-scroll/virtual-scroll.ts

```diff
@@ -528,7 +528,13 @@ export class VirtualScroll implements DoCheck, AfterContentInit, OnDestroy {
   }
 
   private _updateDiffer() {
-    if (isBlank(this._differ) && isPresent(this._records)) {
```

Comment:
Removing `isBlank` check from the current codebase will have the desired effect. If interested, please updated your pull request.